release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,265 @@ toc::[]
+** The `Access` permission will replace the following permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** The `DeploymentExtension` permission will replace the following permissions: `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+** The `Integration` permission will deprecate the following permissions: `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** The `Image` permission will replace the permission `ImageComponent`.
+* Previously, groups were retrieved by using the field `props` (`props.authProviderId`, `props.key`, `props.value`). This field will be replaced by the new `props.id` field. Use the `props.id` field to retrieve groups in the Application Programming Interface (API). Note the following:
+** Retrieval by using the `props` fields will be removed in version a future release.
+** Until removal, retrieval by using the `props` field will work if the result is unambiguous (no more than one group is found with the `props` field).
+* `/v1/cves/suppress` and `/v1/cves/unsuppress` have been deprecated and will be removed in a future release. After these are removed:
+** Use `/v1/imagecves/suppress` and `/v1/imagecves/unsuppress` to snooze and unsnooze image vulnerabilities.
+** Use `/v1/nodecves/suppress` and `/v1/nodecves/unsuppress` to snooze and unsnooze node and host vulnerabilities.
+** Use `/v1/clustercves/suppress` and `/v1/clustercves/unsuppress` to snooze and unsnooze platform (Kubernetes, Istio, and {ocp}) vulnerabilities.
+* Permission `ClusterCVE` is deprecated and will be superseded by the existing permission `Cluster`. 
+* `Label` and `Annotation` search options are deprecated and will be removed in the {product-title-short} 3.73 release. They will be replaced by the search options listed in the following table.
+
+.Search options
+[cols="1,1,1",options="header"]
+|====
+|Resource |Deprecated search option | New search option 
+
+| Node
+| Label
+| Node Label
+
+| Node
+| Annotation
+| Node Annotation
+
+| Namespace
+| Label
+| Namespace Label
+
+| Deployment
+| Label
+| Deployment Label
+
+| ServiceAccount
+| Label
+| Service Account Label
+
+| ServiceAccount
+| Annotation
+| Service Account Annotation
+
+| K8sRole
+| Label
+| Role Binding Label
+
+| K8sRoleAnnotation
+| Annotation
+| Role Binding Annotation
+
+|====
+
+[id="psp-changes"]
+=== PodSecurityPolicy (PSP) objects and Kubernetes 1.25
+
+The following changes were made in {product-title-short} to support the removal of PSP objects in Kubernetes 1.25:
+
+* PSP objects are created by default in installations for backward compatibility with Kubernetes versions prior to 1.25. However, you can manually disable PSP creation by setting `system.enablePodSecurityPolicies = false` in the Helm chart. 
```

Comment:
Based on my read of Helm charts, I think this is best expressed as:

`system.enablePodSecurityPolicies: false`